package.json

```diff
@@ -88,6 +88,9 @@
       "lib/content-data-generator"
     ]
   },
+  "resolutions": {
```

Comment:
so this won't work for a number of reasons: 

- this project is not using yarn so resolutions will not have any effect
- we should not be relying on resolutions to fix issues like this
- it seems like this problem will cause issues after upgrading as mentioned in https://github.com/ember-cli/ember-fetch/issues/547#issuecomment-671885942

can you revert this change, and if it is still causing an issue then we can look at upgrading ember-fetch which seems to have fixed the issue 👍

Reply:
Thanks a lot for reviewing this PR. I've updated ember-fetch version as this issue was still occurring.